packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -532,16 +532,28 @@ module.exports.external = [
   },
   {
     name: "multipleToArray",
-    description: "Returns an array combining all given objects: multipleToArray obj1 obj2 …",
     func: function (...vals) {
-        var combinedArr = [];
-        vals.forEach(function(val) {
-            if (Array.isArray(val)) {
-                combinedArr.push(...val);
-            } else if (val) {
+        const uniqueSet = new Set();
```

Comment:
we are often passing duplicate entries into multipleToArray. Using sets to make sure all return values are unique. Important for determining how many Encounters we actually have in a file